djangocms_versioning/admin.py

```diff
@@ -1,9 +1,17 @@
 from django.apps import apps
 from django.contrib import admin
+from django.contrib.admin.options import IncorrectLookupParameters
+from django.contrib.admin.views.main import ChangeList
+from django.contrib.contenttypes.models import ContentType
 from django.urls import reverse
 from django.utils.html import format_html
 from django.utils.translation import ugettext_lazy as _
 
+from .models import Version
+
+
+GROUPER_PARAM = 'grouper'
```

Comment:
Are we assuming that the grouper fk on the content model will always be called "grouper"? Also if it's a constant then it probably needs to be in constants.py, right? :)

Reply:
This is just a query param name, it's not related to the name of grouper FK field on content model. Might move it to `constants.py`, as it will be used to build "Manage versions" URL.